.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,112 @@
+name: Prepare release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      generate_changelog:
+        required: false
+        default: true
+        type: boolean
+      update_db:
+        required: false
+        default: false
+        type: boolean
+      db_dump_path:
+        required: false
+        type: string
+      migration_command:
+        required: false
+        default: './vendor/bin/phinx migrate'
+        type: string
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+
+      - name: Update package version
+        run: |
+          if [ -f "package.json" ]; then
+            npm pkg set version=${{ inputs.version }}
+          fi
+
+      - name: Update changelog file
+        if: ${{ inputs.generate_changelog == true }}
+        uses: uniquesca/continuous-integration/update-changelog@main
+        with:
+          target_version: ${{ inputs.version }}
+
+      - name: Update DB dump
+        if: ${{ inputs.update_db == true }}
+        run: |
+          php composer.phar config --auth github-oauth.github.com ${{ secrets.ACCESS_TOKEN }}
+          php composer.phar install
+
+      - name: Init database
+        run: |
+          sudo service mysql start
+          mysql -u root -proot --execute="CREATE DATABASE mysql_test;"
+
+      - name: Import DB
+        uses: uniquesca/continuous-integration/mysql-import@main
+        if: ${{ inputs.update_db == true }}
+        with:
+          dump_file_path: ${{ inputs.db_dump_path }}
+          db_name: mysql_test
+
+      - name: Prepare config for DB and install migrations
```

Comment:
Let's remove this step. It's very specific to a calling repo, and has to be done prior to calling this workflow